pages.zh/linux/boltctl.md

```diff
@@ -0,0 +1,28 @@
+# boltctl
+
+> 控制雷电 (thunderbolt) 设备。
+> 更多信息：<https://manned.org/boltctl>.
+
+- 列出已连接并授权的设备：
+
+`boltctl`
+
+- 列出已连接的设备，且包含未授权的设备：
+
+`boltctl list`
+
+- 临时授权一个设备：
+
+`boltctl authorize {{设备UUID}}`
```

Comment:
I think this should be lower case:
```suggestion
`boltctl authorize {{设备uuid}}`
```

Reply:
UUID stands for Universally Unique IDentifier so given that this is an acronym, in English we would keep this as uppercase @navarroaxel. Don't know about Chinese though.

Reply:
@sbrl I understand but we use `snake_lowercase` as variables, in some programming languages like C# the acronym are PascalCased like `XmlDocument` or `Guid` because is the standard for class names. So, I think `UUID` could be misreading.

Reply:
Oh, I see

Ignore my comment then. Let's apply this change.